modules/ve-cx/init/ve.init.mw.CXTarget.js

```
@@ -1110,11 +1120,20 @@ ve.init.mw.CXTarget.prototype.changeContentSource = function (
  *
  * @param {number} sectionNumber
  */
-ve.init.mw.CXTarget.prototype.prefetchTranslationForSection = function ( sectionNumber ) {
+ve.init.mw.CXTarget.prototype.prefetchTranslationForSection = function ( sectionNumber, set_it ) {
```

ESLint

[error] Identifier 'set_it' is not in camel case.

(camelcase)